app/Http/Controllers/HomeController.php

```diff
@@ -58,4 +71,18 @@ public function getUserGroups($email)
 
         return $userGroups;
     }
+
+    public function storeEmployeeLocation(Request $request)
+    {
+        $validatedData = $request->validate([
+            'Centre' => 'required',
```

Comment:
The name should be in small case. Also read about [exist rule in Laravel](https://laravel.com/docs/10.x/validation#basic-usage-of-exists-rule)
```suggestion
            'centre_name' => 'required | exists:office_locations,centre_name',
```